cmd/sabayon/main.go

```diff
@@ -23,6 +24,7 @@ func main() {
 	var email = os.Getenv("ACME_EMAIL")
 	var token = os.Getenv("HEROKU_TOKEN")
 	var appName = os.Getenv("ACME_APP_NAME")
+	wait, _ := strconv.Atoi(os.Getenv("SABAYON_WAIT"))
```

Comment:
This will default to 0 if no config var is set? Something like: https://play.golang.org/p/FHpSligzZh